src/sas/qtgui/MainWindow/UI/MainWindowUI.ui

```diff
@@ -24,7 +24,7 @@
      <x>0</x>
      <y>0</y>
      <width>915</width>
-     <height>26</height>
+     <height>20</height>

```

Comment:
I don't know why that's been changed, would have been done by Qt designer. Does it affect anything?

Reply:
`height 20` is in `release_6.0.0` so probably your UI files didn't get properly updated on pulling main, seeing as `height 26` is there.
I don't know why the difference arose and the menu bar doesn't look that much different, unless compared closely.
![menu_bar_6 0](https://github.com/user-attachments/assets/82bd9ec5-3b1d-47c8-bc27-bd9786456260)

Reply:
I don't think it affects anything. Maybe it automatically snaps back into this geometry because of the QSizePolicy of the Menubar. The vertical size policy for the menubar is 'minimum'.